dnas/sensemaker/zomes/coordinator/sensemaker/src/init.rs

```diff
@@ -1,80 +1,29 @@
+    if let false = Properties::is_community_activator(my_key.clone())? {
         return Ok(InitCallbackResult::Pass);
     }
-    if let Some(config) = prop.config {
-        // check the format of the config passed from Wizard
-        config.clone().check_format()?;
-
-        // create all entries specified in the config
-
-        // dimensions
-        let _dimension_ehs = config
-            .dimensions
-            .clone()
-            .into_iter()
-            .map(|dimension: Dimension| create_entry(&EntryTypes::Dimension(dimension)))
-            .collect::<ExternResult<Vec<ActionHash>>>()?;
-
-        // resource types
-        let _resource_type_ehs = config
-            .resources
-            .clone()
-            .into_iter()
-            .map(|resource| {
-                let converted_resource_type = ResourceType::try_from(resource)?;
-                create_entry(&EntryTypes::ResourceType(converted_resource_type))
-            })
-            .collect::<ExternResult<Vec<ActionHash>>>()?;
-
-        // methods
-        let _method_ehs = config
-            .methods
-            .clone()
-            .into_iter()
-            .map(|method| {
-                let converted_method = Method::try_from(method)?;
-                create_entry(&EntryTypes::Method(converted_method))
-            })
-            .collect::<ExternResult<Vec<ActionHash>>>()?;
 
-        // contexts
-        let _context_ehs = config
-            .contexts
-            .clone()
-            .into_iter()
-            .map(|context| {
-                let converted_context = CulturalContext::try_from(context)?;
-                create_entry(&EntryTypes::CulturalContext(converted_context))
-            })
-            .collect::<ExternResult<Vec<ActionHash>>>()?;
-
-        // create the congif entry and link it off of the community activator
-
-        let converted_config = SensemakerConfig::try_from(config)?;
-        let community_activator_apk: AgentPubKey = prop.community_activator.into();
-        create_entry(&EntryTypes::SensemakerConfig(converted_config.clone()))?;
-        create_link(
-            community_activator_apk,
-            hash_entry(converted_config)?,
-            LinkTypes::CAToSensemakerConfig,
-            (),
-        )?;
-    } else {
-        return Ok(InitCallbackResult::Pass);
+    // create the sensemaker entry from the prop and link it from the community activator
+    let sensemaker_eh = hash_entry(prop.sensemaker_config.clone())?;
+    create_entry(&EntryTypes::SensemakerConfig(prop.sensemaker_config))?;
+    create_link(
+        my_key.clone(),
+        sensemaker_eh.clone(),
+        LinkTypes::CAToSensemakerConfig,
+        LinkTag::new("sensemaker"),
+    )?;
+
+    // for each config, check the format and creat the config entries with the helper function
+    for config in prop.applet_configs {
+        config.clone().check_format()?;
+        create_entries_from_applet_config(config)?;
```

Comment:
Ok so I see that you've consolidated the process of adding config items (resource types, ranges, dimensions, methods, etc.) into the `AppletConfig` type. I don't see any issues with this approach, however, it makes me think about to what extent do we want to distinguish config specifically added because of installing an applet (registering an applet) vs generic config added by the CA either at sensemaker spawning or sometime afterwards that is not for any specific applet. 

Perhaps we would use a keyword as a applet name when adding config not from applet registration, which could be wrapped in a new zome function `add_sensemaker_config`, when a CA wants to, for example, add new dimension. Or something like that. I don't think these are critical, blocking questions right now, but how we answer them will affect how we index/access data from the sensemaker to display in the dashboard.